Sources/GRPCCore/Streaming/RPCWriter+Closable.swift

```diff
@@ -28,6 +28,17 @@ extension RPCWriter {
       self.writer = other
     }
 
+    /// Writes a single element.
+    ///
+    /// This function suspends until the element has been accepted. Implements can use this
```

Comment:
Here and elsewhere: s/Implements/Implementers (sorry, looks like this is copy+pasta from my own typo 😅)